### PR TITLE
EUI-4742 - Mock Judicial Booking API to enable development

### PR DESCRIPTION
**Pull request by tarikdogan-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-4742

### Change description ###

mock implimentation of am/bookings to enable development

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: api/judicialBooking/data/bookings.mock.data.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Bookings } from "judicialBooking/models/booking.model";
```
Comment (paulhowes-HMCTS) [inline api/judicialBooking/data/bookings.mock.data.ts:1]: Can we call this and the corresponding objects booking as opposed to bookings please?

File: api/judicialBooking/index.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { bookings } from './data/bookings.mock.data';
+
+export const getBookings = (proxyReq, req, res) => {
+    // when am/booking url is developed , there will be switch flag implimantation to get responds from url or from mock data
```
Comment (paulhowes-HMCTS) [inline api/judicialBooking/index.ts:4]: implimantation -> implementation

File: api/judicialBooking/models/booking.model.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+export interface Booking {
+    userId: string;
+    appointmentId: string;
+    roleId: string;
+    contract_type_id: string;
```
Comment (paulhowes-HMCTS) [inline api/judicialBooking/models/booking.model.ts:5]: Any reason why we have moved to snake case or is this how the data will be returned from the endpoint?

File: test/nodeMock/ccd/judicialBookings_data.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
```
Comment (paulhowes-HMCTS) [inline test/nodeMock/ccd/judicialBookings_data.js:1]: Looking in the folder most of the files don't have _data prefixed on them, so for consistency can you remove _data from the filename.

Comment (paulhowes-HMCTS) [reply test/nodeMock/ccd/judicialBookings_data.js:1]: Ideally call it judicialBooking.js (singular)

**Review by paulhowes-HMCTS — CHANGES_REQUESTED** [review]
A few minor tweaks, mostly to naming conventions / pluralisation. At the time of this review it is still building.

File: api/judicialBooking/models/booking.model.ts (hunk 3 above)
Comment (tarikdogan-hmcts) [reply api/judicialBooking/models/booking.model.ts:5]: this is how data will return from api.

File: test/nodeMock/ccd/judicialBookings_data.js (hunk 4 above)
Comment (tarikdogan-hmcts) [reply test/nodeMock/ccd/judicialBookings_data.js:1]: this file shouldn't exist, removed .

**Review by paulhowes-HMCTS — DISMISSED** [review]
All queries discussed/addressed/resolved so approving.

**Review by paulhowes-HMCTS — APPROVED** [review]
Re-approving

**Review by adnanakgun — APPROVED** [review]

**Pull request merged**